django-verdant/rca/api/serializers.py

```diff
@@ -154,3 +154,4 @@ def to_representation(self, image):
 class RCAImageSerializer(ImageSerializer):
     original = ImageRenditionField('original', read_only=True)
     thumbnail = ImageRenditionField('max-165x165', read_only=True)
+    rca2019_feed_image = ImageRenditionField('fill-828x472', read_only=True)
```

Comment:
I missed one for this, in some places Nick is using lazy load so could we also add:
```
rca2019_feed_image_small = ImageRenditionField('fill-80x46', read_only=True)
```